PWGHF/treeHF/AliHFJetFinder.h

```diff
@@ -61,6 +61,7 @@ class AliHFJetFinder : public TObject
   std::vector<AliHFJet> GetJets(TClonesArray *array);
   std::vector<AliHFJet> GetMCJets(TClonesArray *array);
   void FindJets(TClonesArray *array, AliAODRecoDecayHF *cand=nullptr, Double_t invmass=0);
+  void FindMCdaughters(TClonesArray *array, AliAODMCParticle *mcpart, std::vector<Int_t>& daughter_vec);
```

Comment:
For consistency, I would suggest calling it `FindMCDaughters`.

Reply:
Thanks, corrected.